### PR TITLE
Disable koji users

### PR DESCRIPTION
**Pull request by ehelms**
Follow on to https://github.com/theforeman/foreman-infra/pull/1625 once the structure there is agreed to to disable inactive Koji users.

### CODE REVIEW

File: koji/vars/users.yml [hunk 1]
```diff
@@ -109,17 +74,38 @@ enabled_users:
...
-      - admin
   - name: zhunting
     permissions:
       - admin
       - build
       - repo
 
 disabled_users: []
+  - name: adprice
+    permissions: []
```
Comment (ekohl) [inline koji/vars/users.yml:85]: It feels odd that you have to specify permissions for a disabled user. Can't we default that to an empty array?

Comment (ehelms) [reply koji/vars/users.yml]: Fair thought, I hard set that in the playbook here https://github.com/theforeman/foreman-infra/pull/1625

File: koji/vars/users.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    permissions:
+      - admin
+  - name: ewoud
+    permissions:
+      - admin
+  - name: jenkins
+    permissions:
+      - build
+      - repo
+  - name: jenkins.pulp
```
Comment (ekohl) [inline koji/vars/users.yml:18]: Is this still used? Or only when Pulp 2 really is EOL?

Comment (ehelms) [reply koji/vars/users.yml:18]: That was my assumption -- @pcreech do you know?

Comment (pcreech) [reply koji/vars/users.yml]: It's the one for "nightly" builds of pulp.  Which, nightly hasn't really worked in a while anyawys.  We can probably kill 'em

**Comment by ehelms** [comment]
Cleaned up and I think it's now an accurate reflection. Once this is merged I plan to go clean up the generated certificates on Koji itself.

File: koji/playbooks/users.yml [hunk 3]
```diff
@@ -11,13 +11,17 @@
         name: "{{ user.name }}"
         state: "enabled"
         permissions: "{{ user.permissions }}"
+        koji: 'koji'
```
Comment (ekohl) [inline koji/playbooks/users.yml:14]: Isn't this default?

**Review by ekohl — COMMENTED** [review]
The list of users to disable looks correct to me. It does look odd to see some users with both `admin` and `build` / `repo` permissions and others don't.

File: koji/playbooks/users.yml (hunk 3 above)
Comment (ehelms) [reply koji/playbooks/users.yml:14]: Nope. https://github.com/ktdreyer/koji-ansible#koji-profiles -- check the first line.

**Comment by ehelms** [comment]
Yea. I updated it so either someone is a fully admin or they have build/repo as I think those are two most common profiles.

File: koji/vars/users.yml [hunk 4]
```diff
@@ -78,48 +39,46 @@ enabled_users:
   - name: kojira.katello.org
     permissions:
       - build
+      - repo
   - name: lzapleta
     permissions:
       - admin
```
Comment (ekohl) [inline koji/vars/users.yml:45]: I recall @lzap wanted to hand over Koji maintenance. Do you want to keep admin access or is it sufficient to have regular build/repo permissions?

**Review by ekohl — APPROVED** [review]
I'm ok with starting here (already cleans up a lot).

**Pull request merged**